R/runDPCoA.R

```diff
@@ -91,7 +95,7 @@ setGeneric("calculateDPCoA", signature = c("x", "y"),
            function(x, y, ...)
                standardGeneric("calculateDPCoA"))
 
-.calculate_dpcoa <- function(x, y, ncomponents = 2, ntop = NULL,
+.calculate_dpcoa <- function(x, y, ncomponents = 2, ntop = NULL, subset.row = subset_row,
```

Comment:
Is this line under 80 characters long? Bioc guidelines https://contributions.bioconductor.org/r-code.html?q=80%20cha#indentation

Reply:
ACK.